koku/api/report/queries.py

```diff
@@ -1031,6 +1034,36 @@ def _set_tag_filters(self, filters):
                 filters.add(q_filter)
         return filters
 
+    def _set_operator_specified_filters(self, filters, operator):
```

Comment:
Can this move to `api.query_filter.QueryFilter` to de-duplicate it from the tag query handler?